doc/style-guide-markdown.md

```diff
@@ -0,0 +1,42 @@
+Style Guide for Markdown
+========================
+
+This extends the [general programming](style-guide-programming.md) portion of the [Style Guide](style-guide.md).
+
+
+* Markdown Formatting
+    * Follow GitHub Flavored Markup.
+    * Paragraphs should be all on one long line.
+    * Avoid HTML markup.
+    * Prefer to use indentation instead of code fences.
+* Headings and Subheadings
+    * Documents start with a heading.  No other place uses a first level heading.
+    * Prefer `====` and `----` for first and second level headings.
```

Comment:
Do you want the `====` to be as long as the content above it?

Something Here
============